### DISCUSSION TITLE
Alarm not arming and status unsuccessful in logs

### DISCUSSION THREAD

**Discussion by adeel719** [tsightler/ring-mqtt#397] [Support Q&A]
I have installed this and can activate the alarm in home assistant dashboard. However I also have Google assistant integration with my HA and can't arm the alarm. In my Google cloud logs I get the below, what does this mean and where can I find further logs?

status: {
isSuccess: false
statusType: "CHALLENGE_NEEDED"
}

**Comment by tsightler** [comment]
This is something you should pursue on the Home Assistant forums vs here as clearly the ring-mqtt to HA portion is working.  I'm not particularly familiar with the HA Google Home integration, but I believe I remember reading that you have to set `secure_devices_pin` configuration value for the Google Assistant config in integration.
